completions/zsh/_brew_services

```diff
@@ -0,0 +1,104 @@
+#compdef brew
+#autoload
+
+__brew_installed_services() {
+  local -a services
+  # services=($(brew services list | awk '{print $1}' | tail -n+2))
+  services=(a b c)
```

Comment:
Whoops - yeah I was testing without anything I wanted to install as a service ;)
Thanks for picking that up